docs/modelsearch.rst

```diff
@@ -412,6 +414,41 @@ MFL support the following model features:
 +---------------+-------------------------------+--------------------------------------------------------------------+
 | LAGTIME       | None                          | Absorption lagtime                                                 |
 +---------------+-------------------------------+--------------------------------------------------------------------+
+| COVARIATE     | `parameter`, `covariate`,     | Covariate effects                                                  |
+|               | `effect`                      |                                                                    |
++---------------+-------------------------------+--------------------------------------------------------------------+
+
+
+.. _mfl_symbols:
+
+Symbols
+~~~~~~~
+
+The MFL supports defining symbols via the syntax `LET(SYMBOL, [...])`.  For
+instance, to declare a list of absorption parameters use `LET(ABSORPTION, KA)`.
+Those symbols can then be referred to when declaring covariate effects via the
+syntax `COVARIATE(@SYMBOL, ...)`. Certain symbols have an automatic definition
+when not manually defined:
```

Comment:
To me this paragraph is a little confusing. If I understand it correctly, you use the LET syntax to say that when we refer to ABSORPTION, KA is the parameter we mean. I was thinking that the order of the paragraph might make it a little confusing for me. The order that feels more logical for me would be first to explain what a symbol is, then write that Pharmpy automatically "decides" the symbols, then the table, and then write that you can also manually declare the different types of parameters.

Reply:
OK. I will rewrite.